patch/config-disable-suspend.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index 4efabb8..587cef3 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -546,7 +546,7 @@ CONFIG_USE_PERCPU_NUMA_NODE_ID=y
+ # Power management and ACPI options
+ #
+ CONFIG_ARCH_HIBERNATION_HEADER=y
+-CONFIG_SUSPEND=y
```

Comment:
Set it to 'n'?

Reply:
It's a Linux standard way to make the config option "n" by commenting it out. I think this is because some old kernel code still can use ```#ifdef CONFIG_OPTION```